agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/VariantRepository.java

```diff
@@ -73,16 +73,4 @@ public List<Allele> getAllelesOfVariant(String variantID) {
 				.collect(Collectors.toList());
 	}
 
-	public GenomeLocation getGenomeLocation(String geneID) {
```

Comment:
Where was this method being called? I don't see a removal of a calling unit. Maybe it was never used?

Reply:
It wasn't being used.

Reply:
Great. Then it should go out.